vale/Buildkite/existence.yml

```diff
@@ -7,3 +7,5 @@ swap:
   whitelist: allowlist
   blacklist: blocklist
   oAuth: OAuth
+  'github\.com': github.com
+  github: GitHub
```

Comment:
This addition to `existence.yml` will catch this in the future (and others like it) but not `github.com` (which appears in a bunch of places.